lib/exclude-private-content-sources-extension.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+module.exports.register = function () {
+  if (this.getVariables().playbook.env.GIT_CREDENTIALS) return
+  this.once('playbookBuilt', function ({ playbook }) {
```

Comment:
This is very cool 👍🏾.

I'm seeing ways that the site content can be used to build all sorts of other stuff.

Reply:
Indeed. I think a next step is to integrate the icon loading directly into Antora so it doesn't have to be done as a separate script. But I think it's best to make that change in a separate PR.